RELEASE_PROCESS.md

```diff
@@ -62,20 +60,58 @@ An exception are PRs open via the Github mergify integration (i.e., backported P
   * When bugs are found, create a PR for `main`, and backport fixes to the release branch.
   * Create new release candidate tags after bugs are fixed.
 * After the team feels the release candidate is mainnet ready, create a full release:
-  * Update `CHANGELOG.md`.
-  * Run `make format` to format the code.
+  * **Note:** The final release MUST have the same commit hash as the latest corresponding release candidate.
   * Create a new annotated git tag in the release branch (follow the [Tagging Procedure](#tagging-procedure)). This will trigger the automated release process (which will also create the release artifacts).
   * Once the release process completes, modify release notes if needed.
 
 ### Changelog
 
-You can obtain the changelog by running:
+For PRs that are changing production code, please add a changelog entry in `.changelog` (for details, see [contributing guidelines](../../CONTRIBUTING.md#changelog)). 
```

Comment:
```suggestion
For PRs that are changing production code, please add a changelog entry in `.changelog` (for details, see [contributing guidelines](./CONTRIBUTING.md#changelog)). 
```

Reply:
IMO it's better to keep md-links relative on local checkout and not referencing different version on github

Reply:
Agreed, yeah that's better